README.md

```diff
@@ -177,6 +177,45 @@ $ mvn clean integration-test -Phit-refresh -Dport=8080
 
 Maven will start the server and you can see it at `http://localhost:8080`.
 
+## Using in servlet app
+
+Create a take with constructor accepting `ServletContext`:
+
+```java
+package com.myapp;
+
+public final class TkApp implements Take {
+  private final ServletContext ctx;
+
+  public TkApp(final ServletContext context) {
```

Comment:
@g4s8 Why exactly is the ``ServletContext`` needed here?

Reply:
@amihaiemil I think the weak part in Takes is a `Front` and `Back` implementations, because it open scokets manually and doesn't support some http features, like `keep-alive` headers and `HTTP/2`, using Takes inside Servlets can solve this - Servlet server (e.g. Tomcat) will control transport layer (open TCP sockets, decide keep them or close, etc) and Takes will control only HTTP - respond to requests.

Reply:
@g4s8 I see; that's a good point, actually :)